fast_llm/layers/vision_encoder/adapter.py

```diff
@@ -0,0 +1,55 @@
+import typing
```

Comment:
This is just a MLP, let's make it one.

Reply:
Main benefit is we get the implementation for free, no new code needed. Flexibility is just a side effect.